base/loading.jl

```diff
@@ -1200,7 +1200,7 @@ function run_extension_callbacks(pkgid::PkgId)
     # take ownership of extids that depend on this pkgid
     extids = pop!(EXT_DORMITORY, pkgid, nothing)
     extids === nothing && return
-    for extid in extids
+    for extid in values(extids)
```

Comment:
extids is an Array, so calling `values` is unnecessary

Reply:
Forgot that pop changes dicts into arrays.

Reply:
It doesn't? The values of that dict are vectors.

Reply:
Yeah , sorry :(. In any case do we have a way to check if a pkg is an extension at this time? To get back the pretty printing?